api-tests/src/steps/ipv-steps.ts

```diff
@@ -131,27 +129,15 @@ When(
       !!reproveIdentity,
       { inheritedIdentityId },
       featureSet,
-      undefined,
     );
   },
 );
 
 When(
-  "I start a new {string} journey with invalid redirect url {string}",
-  async function (
-    this: World,
-    journeyType: string,
-    redirectUrl: string,
-  ): Promise<void> {
+  "I start a new {string} journey with invalid redirect uri",
```

Comment:
Do we need this any more - could we just use the 'normal' start new journey step?

Reply:
I think I kept this separate because for this case, we expect an error to get thrown from `initialiseIpvSession` that we want to catch and keep track of so we can assert against it in a future step, whereas in the 'normal' start new journey step, if an error gets thrown from say `initialiseIpvSession` we'd want it to prevent the subsequent steps from running